### ISSUE TITLE
Next won't render websites and just shows a white screen

### ISSUE THREAD

**Issue by casidiablo** [atlas-engineer/next#611]
New to this browser, which I installed in my nixos box.

When trying to open a webpage, it only shows a blank screen. The website does seem to load: when I move my mouse around I do see in the bottom bar the URLs of the links that should be underneath my pointer. i.e.: the website does seem to render, but it's invisible or just white.

- OS nixos 19.0:
- Kernel: Linux 5.5.7
- Graphics card and driver: nvidia
- Desktop environment / Window manager name+version: i3
- Next version: happens in both 1.4.0 and 1.5.0
- How you installed Next: nix package manager
- Lisp implementation/version: not sure
- WebKitGTK+ | QtWebEngine version: webkitgtk-2.26.4

Output:

```
Next version 1.5.0
 <INFO> [10:56:40] next remote.lisp (initialize-instance :after remote-interface) -
  D-Bus addresses: (#<Unix socket address: @"/tmp/dbus-gQHt1mQs6V">)
 <INFO> [10:56:40] next remote.lisp (initialize-instance :after remote-interface) -
  Bus connection name: :1.25416
 <INFO> [10:56:40] next port.lisp (run-program port) -
  Current directory: /home/cristian/.config/nixpkgs/
 <INFO> [10:56:40] next port.lisp (run-program port) -
  Platform port path: /nix/store/p0mmhi13hf83fxck4366sac9vhcjjrn0-next-gtk-webkit-1.5.0/bin/next-gtk-webkit
 <INFO> [10:56:40] next port.lisp (run-program port) -
  Platform port arguments: NIL
 <INFO> [10:56:40] next port.lisp (run-program port) -
  Platform port log file: /home/cristian/.local/share/next/next-gtk-webkit.log
 <WARN> [10:56:40] next remote.lisp (current-buffer) -
  No active window, picking last active buffer.
 <WARN> [10:56:41] next session.lisp (restore-sexp-session form-fun-4) -
  Session version "1.4.0" differs from current version "1.5.0"
 <INFO> [10:56:41] next session.lisp (restore-sexp-session form-fun-4) -
  Restoring (https://www.wikipedia.org/ https://www.wikipedia.org/
             https://www.wikipedia.org/)
 <WARN> [10:56:41] next history.lisp (restore-sexp-history form-fun-4) -
  Session version "1.4.0" differs from current version "1.5.0"
 <INFO> [10:56:41] next minibuffer.lisp (echo form-fun-6) -
  "Loading global history of 3 URLs."
 <INFO> [10:56:41] next minibuffer.lisp (echo form-fun-6) -
  "Finished loading: https://www.wikipedia.org/."
 <INFO> [10:56:41] next minibuffer.lisp (echo form-fun-6) -
  "Finished loading: https://www.wikipedia.org/."
 <INFO> [10:56:41] next minibuffer.lisp (echo form-fun-6) -
  "Finished loading: https://www.wikipedia.org/."
 <INFO> [10:56:41] next minibuffer.lisp (echo form-fun-6) -
  "Loading: https://www.wikipedia.org/."
 <INFO> [10:56:41] next minibuffer.lisp (echo form-fun-6) -
  "Loading: https://www.wikipedia.org/."
 <INFO> [10:56:41] next minibuffer.lisp (echo form-fun-6) -
```

**Comment by Ambrevar** [comment]
Does it work if you export WEBKIT_DISABLE_COMPOSITING_MODE=1 beforehand?
Can you test master?

**Comment by casidiablo** [comment]
It works fine if I set `WEBKIT_DISABLE_COMPOSITING_MODE=1` :+1: 

Not sure whether to close this, and open a pr in nixos to fix it in the package definition, or if this can be fixed on your side.

**Comment by Ambrevar** [comment]
This is probably going to be fixed in Next 2.0, you can confirm by
testing master ;)
